Example/ViewController.swift

```diff
@@ -360,6 +358,12 @@ class ViewController: UIViewController {
     }
 }
 
+extension ViewController: UnimplementedLogging {
+    var delegateIdentifier: String {
+        return "navigationMapViewDelegate+navigationViewControllerDelegate+voiceControllerDelegate+waypointConfirmationViewControllerDelegate"
```

Comment:
The intent of these identifiers is to make the console spew more explicit about the protocol that isn’t fully implemented. However, I was concerned that this COM-reminiscent approach would be fragile – imagine copying a conforming class and forgetting to update the identifier, or making the class conform to an additional protocol without updating the identifier. So @JThramer is going to modify `logUnimplemented(level:function:)` to take an explicit protocol name as an argument. This way the console spew will state the precise protocol that the unimplemented method belongs to.

Reply:
Indeed. And it's done as of 38514ba.